app/controllers/reviews_controller.rb

```diff
@@ -17,7 +22,7 @@ def edit
   def update
     review = Review.find(params[:review_id])
     if review.update(review_params) == false
-      flash[:notice] = "A required field is missing; please fill out all fields!"
+      flash[:notice] = "Please fill out all required fields."
```

Comment:
Updated your flash message to stay consistent throughout the site. We can change this if you would like.